tests/models/test_cpu.py

```diff
@@ -494,6 +496,7 @@ def train_dataloader(self):
     )
 
     model = BpttTestModel(**hparams)
+    model.example_input_array = None
```

Comment:
can we also add a test here, where this is set? TO test that this actually works?

Reply:
Done :)